_content/monographs/bodipakkhiyadipani_sayadaw-ledi.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Bodipakkhiyadipani: The Manual of the Factors Leading to Enlightenment"
+authors:
+  - "Ledi Sayadaw"
+translator: :"U Sein Nyo Tun"
+external_url: "https://dhamma-library.cdn.winmetta.org/14.%20Rear-Buddhist-Books-2/Bodhipakhiyadipani.pdf"
```

Comment:
Let's move this to `source_url` and use the .html version of the Wheel link as the external_url